### ISSUE TITLE
Exception in channel_establishment_flow: StorageReadWriteError(OSError(FileNotFoundError(2, 'No such file or directory')))

### ISSUE THREAD

**Issue by HamishMacEwan** [spesmilo/electrum#7733]
Following an upgrade to 4.2.0 (though I haven't opened a channel in some time) the subject error message appeared while attempting to open a channel.

The file in question is described here:

`FileNotFoundError: [Errno 2] No such file or directory: 'm:\\⚡.backup.tmptest.13296`

There is no M: drive on the system at present.

SOLVED: I found an entry in …\Electrum\config    "backup_dir": "M:/" and after fixing that to reflect the current structure:

![image](https://user-images.githubusercontent.com/388061/159833185-22016269-fb01-4270-810a-79f30cda7cbb.png)

There doesn't seem to be an option in Preferences to set the backup directory, and a more informative error would be helpful.
 
A more complete error report:
```
20220324T023456.117920Z |    ERROR | lnpeer.Peer.[LNWallet, 02a7e56ed6-a1024956] | Exception in channel_establishment_flow: StorageReadWriteError(OSError(FileNotFoundError(2, 'No such file or directory')))
Traceback (most recent call last):
  File "electrum\util.py", line 1700, in test_read_write_permissions
FileNotFoundError: [Errno 2] No such file or directory: 'm:\\⚡.backup.tmptest.13296'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "electrum\storage.py", line 67, in __init__
  File "electrum\util.py", line 1706, in test_read_write_permissions
OSError: [Errno 2] No such file or directory: 'm:\\⚡.backup.tmptest.13296'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "electrum\util.py", line 1160, in wrapper
  File "electrum\lnpeer.py", line 660, in wrapper
  File "electrum\lnpeer.py", line 856, in channel_establishment_flow
  File "electrum\lnworker.py", line 1030, in add_new_channel
  File "electrum\wallet.py", line 328, in save_backup
  File "electrum\storage.py", line 69, in __init__
electrum.storage.StorageReadWriteError: [Errno 2] No such file or directory: 'm:\\⚡.backup.tmptest.13296'
```

**Comment by ecdsa** [comment]
why was it set to m:// in the first place? was it by you?

**Comment by SomberNight** [comment]
same as https://github.com/spesmilo/electrum/issues/7705
I suspect these users used the `Save backup` feature at one point, and then forgot about it.
The way it works is not so intuitive: it is not clear that using it has consequences outside creating a one-time backup file.

**Comment by ecdsa** [comment]
fixed in a15dac2b8cf15d0bfd6cb50f5c83a69b30897ffb
